### PR TITLE
Fix issue #308 (Period Selector)

### PR DESCRIPTION
**Pull request by Afredon**
Just something that went wrong during a refactoring

### CODE REVIEW

File: src/assets/js/app.js [hunk 1]
```diff
@@ -150,6 +150,7 @@
                 .on(sc.event.dataPeriodChange, function(period) {
                     sc.model.selectedPeriod = period.option;
                     dataInterface(sc.model.selectedPeriod.seconds);
+                    render();
```
Comment (jleft) [inline src/assets/js/app.js]: Do we need to render here?

**Comment by jleft** [comment]
Looks good to me, thanks @Afredon :+1:

File: src/assets/js/app.js (hunk 1 above)
Comment (Afredon) [inline src/assets/js/app.js]: as explained above, we don't as dataInterface will trigger rendering through the event chain (twice I think), but it feels like side effect at the moment as we call render() on the other event (dataProductChange).

Comment (Afredon) [inline src/assets/js/app.js]: I expect once refactorings are done we will be in position to decide exactly when render() is called.

**Pull request merged**